instrumentation-genai/opentelemetry-instrumentation-openai-v2/src/opentelemetry/instrumentation/openai_v2/utils.py

```diff
@@ -225,8 +225,10 @@ def get_llm_request_attributes(
         service_tier if service_tier != "auto" else None
     )
 
-    # filter out None values
-    return {k: v for k, v in attributes.items() if v is not None}
+    # filter out None values and NOT_GIVEN values
+    return {
+        k: v for k, v in attributes.items() if v is not None and v != NOT_GIVEN
```

Comment:
I think the check should be `isinstance(NotGiven)` since there's both `NOT_GIVEN` and `not_given`